repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -426,7 +426,10 @@ def filter_out_pkgs_in_blacklisted_repos(to_install):
         del to_install[pkg]
 
     if blacklisted_pkgs:
-        report_skipped_packages('packages will not be installed due to blacklisted repositories:',
+        # TODO: Describe why they should be installed in the first place (replacement, etc.)
+        report_skipped_packages('packages will not be installed because they are available only in '
+                                'repositories that are either disabled or optional on RHEL8. '
+                                'The list of these packages (repositories: {}):'.format(', '.join(blacklisted_repos)),
```

Comment:
Why not to use the message that we've crafted nicely here: https://github.com/oamg/leapp-repository/pull/519/files#diff-a7492401bb654d7516a25e720ece7239R411?

Reply:
indeed. will fix this